src/components/gutter/gutter.ts

```diff
@@ -110,6 +119,55 @@ export class Gutter {
   async layoutItems () {
     if (this.layout) {
       this.layout.layout([...this.items()]);
+      this.layoutComplete.emit();
+    }
+  }
+
+  /**
+   * Activates the item logically after the currently active item. The activated item's `active` property will be set to true. Returns the
+   * activated item. If there is no currently active active, the top-most item is activated. If the currently active item is
+   * the bottom-most item in the gutter, then the top-most item will be activated.
+   */
+  @Method()
+  async activateNextItem () {
+    if (this.layout) {
```

Comment:
An edge case that is there are no items in the gutter at all.

Reply:
if that is the case then should it not fail loudly?

Reply:
to accommodate this edge case, the function will return null if there is less than 2 `la-gutter-item`s